src/debugger/titaniumDebugConfigurationProvider.ts

```diff
@@ -3,7 +3,7 @@ import * as vscode from 'vscode';
 import * as which from 'which';
 import { UserCancellation } from '../commands';
 import { selectPlatform } from '../quickpicks/common';
-import { AppBuildTaskDefinitionBase } from '../tasks/buildTaskProvider';
+import { AppBuildTaskDefinitionBase, AppBuildTask } from '../tasks/buildTaskProvider';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/debugger/tita...
  
  
  DangerID: danger-id-Danger;
  File: src/debugger/titaniumDebugConfigurationProvider.ts;
  Line: 6;
-->

- :warning: src/debugger/titaniumDebugConfigurationProvider.ts line 6 – 'AppBuildTask' is defined but never used. (@typescript-eslint/no-unused-vars)